WordPress/Classes/Utility/ReachabilityUtils+OnlineActions.swift

```diff
@@ -35,4 +32,21 @@ extension ReachabilityUtils {
                 return notification.userInfo?[Foundation.Notification.reachabilityKey] as? Bool == true
         })
     }
+
+    /// Shows a generic "no internet connection" Notice to the user.
```

Comment:
I personally think that the method name explains this well and there is no need for a comment. What is the purpose of this comment?

Reply:
@yaelirub I felt like I needed a "brief" description so I can add the explanation below: 

```swift
    /// We use a Snackbar instead of a literal Alert because, for internet connection errors,
    /// Alerts can be disruptive.
```

But I guess I could move that to the `func` body instead. Unless you think that part is also not necessary?